lib/index.ts

```diff
@@ -396,6 +406,24 @@ export const getSdk = function ($opts?: SdkOptions) {
 	 */
 	sdk.request = request;
 
+	/**
+	 * @summary Balena utils instance
+	 * @member {Object} utils
+	 * @public
+	 * @memberof balena
+	 *
+	 * @description
+	 * The utils instance used internally. This should not be necessary
+	 * in normal usage, but can be useful to handle some specific cases.
+	 *
+	 * @example
+	 * balena.utils.mergePineOptions(
+	 *  { $expand: { device: { $select: ['id'] } } },
+	 *  { $expand: { device: { $select: ['name'] } } },
+	 * );
+	 */
+	sdk.utils = utils;
```

Comment:
Given that `util/index.ts` loads all utils that the SDK has, maybe it would make sense lazy load this property like we do for the models. Otherwise the individual file loading would slow down the CLI's startup time.
```suggestion
		Object.defineProperty(sdk, 'utils', {
			enumerable: true,
			configurable: true,
			get() {
				const { mergePineOptions } = require('./util') as typeof import('./util');
				return { mergePineOptions };
			},
		});
```
On top of (but we can postpone for later) we could make `util/index.ts` only include the exported functions and have the SDK source (& tests) do exact path imports for anything they need.